src/adapters/expressErrorHandler.ts

```diff
@@ -16,24 +18,16 @@ import H5pError from '../helpers/H5pError';
 export default (languageOverride: string | 'auto' = 'auto') => {
     return async (
         err: Error | H5pError | AggregateH5pError,
-        req: any,
-        res: any,
-        next: any
+        req: IRequestWithTranslator,
+        res: Response,
+        next: NextFunction
     ): Promise<void> => {
         let statusCode = 500;
         let statusText = '';
         let detailsList;
         let clientErrorId = '';
 
         if (err instanceof H5pError) {
-            if (
```

Comment:
Was this removed deliberately? I think we still need it to switch the language to the current user's settings. I have already wondered whether it might not be better to have the implementation call changeLanguage and remove the language override. Any thoughts on this?

If we remove the languageOverride, the currying function in l. 16 must be removed as well, as it doesn't make any sense otherwise.

Reply:
It was a mistake while fixing the merge-conflict.
I will correct this.